### PR TITLE
Helm charts: fixes&compat. fixes, still most from known issues remain

### PR DESCRIPTION
**Pull request by ReSearchITEng**
adopt k8s 1.6 also, added limits for node exporter, fixed build script, made some files from readOnly to readwrite so chown will not fail, added addnotation as per issue 573, added debug option, exposed the ceph_init image, changed NETWORK_AUTO_DETECT to 4 in statefull set (as per one issue)
It has been tested, things improved, but still most of the already reported issues remain, including lack of compatibility with the new images...

### CODE REVIEW

File: examples/helm/build-all.sh [hunk 1]
```diff
@@ -8,7 +8,9 @@ function build_charts () {
   list_dirs | while read CHART; do
     if [ -f ${CHART}/Makefile ]; then make -C ${CHART}; fi
     if [ -f ${CHART}/requirements.yaml ]; then helm dep up ${CHART}; fi
-    helm package ./${CHART}
+    if [ -f ${CHART}/Chart.yaml ]; then 
+      helm package ./${CHART};
```
Comment (leseb) [inline examples/helm/build-all.sh]: what's this `;` at the end of the line?

**Comment by GregMeno** [comment]
jenkins lint

**Comment by GregMeno** [comment]
jenkins lint

**Comment by GregMeno** [comment]
jenkins lint

File: examples/helm/build-all.sh (hunk 1 above)
Comment (ReSearchITEng) [inline examples/helm/build-all.sh]: Initially I've put it inline, like the ones above. Afterwards made it more visible, for people to understand this is the main purpose of running this build as well as a "provision" for future addition like else echo "no chart has been found under folder $CHART.

File: examples/helm/ceph/values.yaml [hunk 2]
```diff
@@ -136,6 +143,13 @@ resources:
     limits:
       memory: "1Gi"
       cpu: "1000m"
+  exporter:
```
Comment (ReSearchITEng) [inline examples/helm/ceph/values.yaml]: I do not have any knowledge what would be the best values for exporters, I've put some values as the installation is failing without it.

**Comment by ReSearchITEng** [comment]
@GregMeno , @leseb : is jenkins' lint working fine on templatized yaml files ?

**Comment by GregMeno** [comment]
@ReSearchITEng Looks like I need to tweak the filter to exclude yaml from shellcheck, I'll do that and re-run

**Comment by GregMeno** [comment]
jenkins lint

**Comment by GregMeno** [comment]
@ReSearchITEng linting is fixed, thank you

**Pull request merged**